openmetadata-ui/src/main/resources/ui/src/pages/AddQueryPage/AddQueryPage.component.tsx

```diff
@@ -227,9 +227,7 @@ const AddQueryPage = () => {
                   <SchemaEditor
                     className="custom-query-editor query-editor-h-200 custom-code-mirror-theme"
                     mode={{ name: CSMode.SQL }}
-                    options={{
-                      readOnly: false,
-                    }}
+                    showCopyButton={false}
```

Comment:
@Ashish8689 , I think we can have default option as a false in the editor, so we only have to pass this prop if we want to make it true, wdyt?

Reply:
so, the idea was to have the copy options as always where ever we are using editor and only some places we don't want that copy button. That the reason we have set by default as true and based on your requirement u can set it to false.